bug: interlink.sock overriden when using github oidc provider

# Short Description of the issue

https://github.com/interTwin-eu/interLink/blob/main/cmd/installer/templates/interlink-install.sh#L128
the above line overrides the socket created by the interlink binary which is then consumed by the oauth2-proxy, removing this will ensure that there are no socket binding issues when starting the interlink process.

## Steps to reproduce

Use github oidc provider during your setup.

## Logs, stacktrace, or other symptoms

```
Every 1.0s: cat logs/interlink.log                                                                                           docker: Tue Oct 29 21:05:50 2024

time="2024-10-29T21:03:36+05:30" level=info msg="Loading InterLink config from /home/ubuntu/.interlink/config/InterLinkConfig.yaml"
time="2024-10-29T21:03:36+05:30" level=info msg="{unix:///home/ubuntu/.interlink/interlink.sock 0 unix:///home/ubuntu/.interlink/plugin.sock 0 true true fals
e ~/.interlink}"
time="2024-10-29T21:03:36+05:30" level=info msg="interLink version: 0.3.1-patch2"
time="2024-10-29T21:03:36+05:30" level=error msg="dial unix /home/ubuntu/.interlink/plugin.sock: connect: no such file or directory"
time="2024-10-29T21:04:06+05:30" level=error msg="dial unix /home/ubuntu/.interlink/plugin.sock: connect: no such file or directory"
time="2024-10-29T21:04:36+05:30" level=error msg="dial unix /home/ubuntu/.interlink/plugin.sock: connect: no such file or directory"
time="2024-10-29T21:05:06+05:30" level=error msg="dial unix /home/ubuntu/.interlink/plugin.sock: connect: no such file or directory"
panic: listen unix /home/ubuntu/.interlink/interlink.sock: bind: address already in use

goroutine 1 [running]:
main.main()
        /home/runner/work/interLink/interLink/cmd/interlink/main.go:122 +0xfcc
```

# Summary of proposed changes

remove the line of code mentioned above

--- a/cmd/installer/templates/interlink-install.sh
+++ b/cmd/installer/templates/interlink-install.sh
@@ -125,7 +125,6 @@ start() {
       echo $! > $HOME/.interlink/oauth2-proxy.pid
       ;;
     github)
-      touch  $HOME/.interlink/interlink.sock
       $HOME/.interlink/bin/oauth2-proxy \
           --client-id {{.OAUTH.ClientID}} \
           --client-secret {{.OAUTH.ClientSecret}} \

